static/js/compose_validate.js

```diff
@@ -491,6 +514,21 @@ export function validate() {
         return false;
     }
 
+    if (message_content.length > page_params.max_message_length) {
+        compose_error.show(
+            $t_html(
+                {defaultMessage: "Message too long: {text_length}/{max_length}"},
+                {
+                    text_length: message_content.length,
+                    max_length: page_params.max_message_length,
+                },
+            ),
+            $("#compose-textarea"),
+        );
```

Comment:
I think we don't need this error message; the red box and red warning tell you what you did wrong, and it's kinda ugly :).  So I just removed the error message part and replaced it with a comment explaining that reasoning.